netci.groovy

```diff
@@ -3,15 +3,26 @@ import jobs.generation.Utilities
 def project = GithubProject
 def branch = GithubBranchName
 def isPr = true
-def containerOs = 'Debian'
-def newJobName = Utilities.getFullJobName(project, containerOs, isPr)
+def platformList = ['Ubuntu16.04:Debian', 'Windows_2016:WindowsServerCore']
 
-def newJob = job(newJobName) {
-    steps {
-        shell("./build-and-test.sh")
-    }
-}
+platformList.each { platform ->
+	def(hostOS, containerOS) = platform.tokenize(':')
+	def newJobName = Utilities.getFullJobName(project, containerOS, isPr)
```

Comment:
I'd prefer `hostOS`, `containerOS`, and `isPr` to follow the same casing convention.